client/extensions/woocommerce/state/ui/promotions/actions.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Internal dependencies
+ */
+import {
+	WOOCOMMERCE_PROMOTIONS_PAGE_SET,
+} from 'woocommerce/state/action-types';
+
+export function setPromotionsPage( siteId, currentPage, perPage = 10 ) {
```

Comment:
In https://github.com/Automattic/wp-calypso/pull/18291/files#diff-eabd23a4dc892f979420ac4ab8ff97b8R21 we have a default of `30` for perPage. Why `10` here?

Reply:
Products has a default of 10, I was following there, but I'll change it to 30 here.

Reply:
Oh, actually the 30 is for loading products and coupons efficiently and doesn't directly relate to the pages. I'm following the Products code, which has 10 per page by default.